.goreleaser.yml

```diff
@@ -193,6 +189,8 @@ dockers:
     - "tykio/tyk-plugin-compiler:{{ .Tag }}"
     - "tykio/tyk-plugin-compiler:v{{ .Major }}.{{ .Minor }}{{.Prerelease}}"
   build_flag_templates:
+    - "--build-arg=branch={{ .Branch }}"
```

Comment:
This line and 193 are not required.

Reply:
LINE 193:
- "--build-arg=TYK_GW_TAG={{ .Tag }}"

Plugin compiler image still relies on that ARG, so not gonna remove it.